docs/END_USER_DOCUMENT.md

```diff
@@ -534,8 +546,20 @@ The table above includes only synchronous clients, and the code samples in this
 
 All Mocca classes mentioned in the table above work as a wrapper containing a default instance of the HTTP client. If the application needs the HTTP client to be configured in a certain manner, there are two ways to achieve that:
 
-1. Mocca client builder could be used to do so, providing a common API regardless of the type of HTTP client used.
-1. A custom instance of the HTTP client could be provided as a constructor parameter to the Mocca HTTP client wrapper.
+1. Mocca client builder could be used to do so, providing a common API regardless of the type of HTTP client used. All currently supported Mocca HTTP clients allow this type of configuration, except the *JAX-RS 2 client*.
+1. A custom instance of the HTTP client (already configured as desired) could be provided as a constructor parameter to the Mocca HTTP client wrapper. This is currently the only way to customize the HTTP client configuration when using a *JAX-RS 2 client*.
```

Comment:
@crankydillo is the JAX-RS 2 client the only one that can use this feature at the moment? I was not sure if Mocca/Feign would honor the configuration set directly in an OkHttp client for example. If Mocca/Feign would not honor that, then this paragraph has to be changed to make that clear.

Reply:
By the way, I believe the answer to that question is yes, but want to double check.

Reply:
For JAX-RS, any timeouts configured in the JAX-RS client API will be respected.  For all other clients, the timeouts configured by builder will be what's used _or_ feign's defaults for these.  I didn't verify this overriding via tests, but we could add it.

This reminds me that I forgot to bring this up about feign's defaults.  IMO, we should have those be Mocca-based defaults, which we'd document.

Reply:
So, the answer to that question is yes. I will push another commit adjust all of this.